packages/docker-engine/0001-bottlerocket-privileged-shouldn-t-disable-SELinux.patch

```diff
@@ -0,0 +1,56 @@
+From 61add2d434d93cfa43a04f259b57b85f8cd99e8f Mon Sep 17 00:00:00 2001
+From: Samuel Karp <skarp@amazon.com>
+Date: Fri, 7 Aug 2020 10:24:15 -0700
+Subject: [PATCH] bottlerocket: privileged shouldn't disable SELinux
+
+In Bottlerocket, the SELinux policy prevents writes to unlabeled
+filesystems.  Disabling SELinux labeling for privileged containers
```

Comment:
nit: "prevents writes to the files underlying the root filesystem by all processes except container runtimes."

Might be other ways to word this. It's not that the filesystem is unlabeled, it's that all the layer files have a highly restricted label (`cache_t`) which only runtimes have permission to alter.

In the absence of a context override, the root filesystem on the upper layer has the same restricted label as the lower layers.

Reply:
Done